release_creation/bundle/download.sh

```diff
@@ -16,4 +16,4 @@ python -m pip download \
 python -m pip download \
  --progress-bar off \
  --dest $base_dir \
- wheel
+ wheel hatchling
```

Comment:
Starting at 1.8, we need hatchling as a build dependency. Since we install without an index later in, we need to include it up front, as we do with `wheel`.